src/components/sidebar/helpers/add-nav-item-meta-data.ts

```diff
@@ -56,7 +56,8 @@ export const addNavItemMetaData = (
 			// Found an `EnrichedLinkNavItem` object
 			if (item.hasOwnProperty('fullPath')) {
 				const itemPath = (item as EnrichedLinkNavItem).fullPath
-				const isActive = !foundActiveItem && itemPath === currentPath
+				const isActive =
+					!foundActiveItem && itemPath.replace(/\/$/, '') === currentPath
```

Comment:
Fixes an issue where items with a trailing slash in their `fullPath` were not matched, even though they're equivalent (eg when comparing `fullPath` of `/terraform/cloud-docs/` with `currentPath` of `/terraform/cloud-docs`)